### ISSUE TITLE
webadminInclude filters in URL

### ISSUE THREAD

**Issue by paulmillar** [dCache/dcache#3709]
Monitoring pages, such as the "Disk Space Usage" / "Pool Usage" page provide filters that result in the page showing only a subset of information; e.g., selecting all pools running within a specific domain.  These filters work client-side (i.e., within the browser), with dCache webadmin knowing nothing about them.

From a post in user-forum, the admin wishes to integrate these filtered pages in some unspecified home-made monitoring system.  My guess is that this uses iframes to include pages within some overview page.  The admin also expressed a desire to share filtered pages with others.

These two requirements would be solved in the page accepted a query-parameter that allowed the URL to "preload" the filters.

**Comment by paulmillar** [comment]
Note that doing this could also solve the reload problem: currently, any filters are lost when the page is refreshed.

**Comment by alrossi** [comment]
The disconnect between the javascript filters and the backend in webadmin is a problem I have addressed before without achieving total success.   It is basically a mess.  What should actually be done is something like what I just did in the frontend:  all the filtering should be on the backend.  

I am not sure how to do what you propose here, I would have to study it.  But from my point of view, this is very low priority, especially in view of the fact that we would like eventually to get rid of this hacky interface.   So spending more time on enhancements does not seem to be the thing to do.

**Comment by paulmillar** [comment]
Agreed -- This issue exists mostly to capture the discussion on user-forum.  There's obviously a desire for this kind of feature.
